src/main/java/teammates/logic/core/AccountsLogic.java

```diff
@@ -225,10 +218,6 @@ private void confirmNotRejoiningUsingDifferentKey(
 
     }
 
-    /**
-     * @throws JoinCourseException if the instructor has already joined this
```

Comment:
Two reasons:
1. The method name has already suggested what it is going to do.
1. If this remains, `JavadocStyle` will complain because the description is empty, and I can't really craft a document that doesn't make the `@throws` description redundant.